sdks/python/.pylintrc

```diff
@@ -143,6 +147,7 @@ disable =
   unnecessary-pass,
   unneeded-not,
   unsubscriptable-object,
+  unspecified-encoding,
```

Comment:
Some of these may be useful to apply eventually. `unspecified-encoding` in particular we should probably fix. Since Beam runs on distributed worker environments that may have surprising default encodings, it's good for our code to be explicit about encodings.

We could add a TODO and file a follow-up jira if it's too much to take on in this PR though.

Reply:
Jira [BEAM-12992](https://issues.apache.org/jira/browse/BEAM-12992) created to follow up `unspecified-encoding` warning